### ISSUE TITLE
App crash when displaying the payment bottom sheet

### ISSUE THREAD

**Issue by ArnaudSpanneut** [stripe/stripe-react-native#1544]
**Describe the bug**

Hi there 👋

Our application crashes when the bottom sheet to enter the card information is displayed.

The error is the same as the one described on the `stripe-android` repo: https://github.com/stripe/stripe-android/issues/7255

```
No static method ModalBottomSheetLayout-BzaUkTc(Lkotlin/jvm/functions/Function3;Landroidx/compose/ui/Modifier;Landroidx/compose/material/ModalBottomSheetState;Landroidx/compose/ui/graphics/Shape;FJJJLkotlin/jvm/functions/Function2;Landroidx/compose/runtime/Composer;II)V in class Landroidx/compose/material/q1; or its super classes (declaration of 'androidx.compose.material.q1' appears in /data/app/~~WMM8uVRYsRdRvilMtKxQRg==/com.alanmobile-hasOPjDXSCrtc_TG_4jUNg==/base.apk)
```

**To Reproduce**
Steps to reproduce the behavior (I'm not sure it'll work as the setup):
1. Install the last version of the stripe-react-native library
2. Install the version 6.0.1 for @intercom/intercom-react-native
3. Setup up both libraries
4. Go through the payment
5. The app is crashing

**Expected behavior**

The bottom sheet is displayed when trying to go through the payment flow

**Smartphone (please complete the following information):**
 - Device: All
 - OS: Andoid
 - Browser N/A
 - Version: [0.33.0](https://github.com/stripe/stripe-react-native/blob/master/CHANGELOG.md#0330---2023-09-22)

**Comment by xseignard** [comment]
tl;dr: to fix the above issue we need a `@stripe/stripe-react-native` version that relies on `stripe-android@20.32.+`

**Comment by xseignard** [comment]
Which is fixed with `@stripe/stripe-react-native@0.34.0` as seen [here](https://github.com/stripe/stripe-react-native/blob/v0.34.0/android/gradle.properties#L3)

This issue can be closed.
